extra/server-discovery/pkg/es/reindex/reindex.go

```diff
@@ -744,3 +782,129 @@ func (ri *reIndexer) Watch(ctx context.Context) {
 		}
 	}()
 }
+
+func (ri *reIndexer) processResource(source *bytes.Buffer) error {
+	var data map[string]interface{}
+	if err := json.Unmarshal(source.Bytes(), &data); err != nil {
+		return fmt.Errorf("Error unmarshaling: %v\n", err)
```

Comment:
```suggestion
		return fmt.Errorf("error unmarshaling: %v\n", err)
```

Error strings should start with lower case